### ISSUE TITLE
 Domoticz crases with MQTT

### ISSUE THREAD

**Issue by loglux** [domoticz/domoticz#2903]
Domoticz crashes getting a connection with mosquitto:

Domoticz(pid:5964, tid:5978('MQTT')) received fatal signal 11 (Segmentation fault)

It was time after time at least once per month and worked fine again after the restart but know I cannot start Domoticz anymore. I even install a different system, changing ubuntu based distro to Debian based one. And in the new environment, I'm getting the same error message and Domoticz crashing

**Comment by loglux** [comment]
fixed the title

**Comment by loglux** [comment]
Dec 06 20:48:34 housespririt domoticz[5964]: Domoticz(pid:5964, tid:5978('MQTT')) received fatal signal 11 (Segmentation fault)
Dec 06 20:48:34 housespririt domoticz[5964]: siginfo address=(nil), address=0xb6d0a2cb
Dec 06 20:48:34 housespririt domoticz[5964]: Failed to start gdb, will use backtrace() for printing stack frame

**Comment by rwaaren** [comment]
Can you post this on the forum and complete with more information like version / hardware / locally compiled or binary from domoticz.com. When it started etc? 
Maybe others active on the forum can help you or comment if they run into the same.

**Comment by loglux** [comment]
I've checked mosquitto log
`root@housespririt:~# sudo tail /var/log/mosquitto/mosquitto.log
1544129288: mosquitto version 1.4.10 (build date Wed, 17 Oct 2018 19:03:03 +0200) starting
1544129288: Config loaded from /etc/mosquitto/mosquitto.conf.
1544129288: Opening ipv4 listen socket on port 1883.
1544129288: Opening ipv6 listen socket on port 1883.
1544129298: New connection from 192.168.10.131 on port 1883.
1544129298: New client connected from 192.168.10.131 as DVES_3B463D (c1, k15, u'DVES_USER').
1544129314: New connection from ::1 on port 1883.
1544129314: New client connected from ::1 as mosq_Ks2j`R5`Wf<caLh?O8 (c1, k60).
1544129314: Socket error on client mosq_Ks2j`R5`Wf<caLh?O8, disconnecting.
1544129319: New connection from 192.168.10.160 on port 1883.
root@housespririt:~#`

**1544129314: New client connected from ::1 as mosq_Ks2j`R5`Wf<caLh?O8 (c1, k60).
1544129314: Socket error on client mosq_Ks2j`R5`Wf<caLh?O8, disconnecting.**

Seems the Espurna flashed POW2 has caused the issue

However, It may be possible incorrectly created a switch or power counter in Domoticz, when I registered the device, is a root of the problem. So, presently I turn off the POW2 and try to create new IDX
